mllib/src/test/scala/org/apache/spark/ml/regression/AFTSurvivalRegressionSuite.scala

```diff
@@ -400,6 +400,17 @@ class AFTSurvivalRegressionSuite
     val trainer = new AFTSurvivalRegression()
     trainer.fit(dataset)
   }
+
+  test("SPARK-19234: Fail fast on zero-valued labels") {
+    val dataset = spark.createDataFrame(Seq(
+         (1.218, 1.0, Vectors.dense(1.560, -0.605)),
+         (0.000, 0.0, Vectors.dense(0.346, 2.158)), // ← generates error; zero labels invalid
+         (4.199, 0.0, Vectors.dense(0.795, -0.226)))).toDF("label", "censor", "features")
+    val aft = new AFTSurvivalRegression()
+    intercept[SparkException] {
```

Comment:
it's recommended to verify the error message using withClue, eg:
withClue("label of AFTPoint must be positive") {
     intercept[SparkException] { 
        aft.fit(dataset)
     }
}